packages/office-ui-fabric-react/src/components/ColorPicker/ColorRectangle/ColorRectangle.base.tsx

```diff
@@ -103,19 +107,19 @@ export class ColorRectangleBase extends BaseComponent<IColorRectangleProps, ICol
       color: newColor
     });
 
+    if (onChange) {
+      onChange(ev, newColor);
+    }
+
     if (onSVChanged) {
       onSVChanged(newColor.s, newColor.v);
     }
+
     ev.preventDefault();
     ev.stopPropagation();
   };
 
   private _onMouseUp = (ev: React.MouseEvent<HTMLElement>): void => {
     this._events.off();
-
-    this.setState({
-      isAdjusting: false,
-      origin: undefined
```

Comment:
just curious, what did these do before?

Reply:
They were set but never accessed anywhere. Probably artifacts of some previous implementation.